news/_posts/2016-11-16-kernel-release-cycle-17.md

```diff
@@ -0,0 +1,34 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 17"
+---
+
+This is a bug fix release. There are a couple of motor driver bug fixes. We
+have also changed the default configuration for the rtl8192cu driver so that
```

Comment:
Can we explain what this series of numbers and letters means? Preferably mention Wi-Fi dongles somewhere.